test/models_test.exs

```diff
@@ -2,13 +2,13 @@ defmodule KazanModelsTest do
   use ExUnit.Case
 
   alias Kazan.Models
-  alias Kazan.Models.Api.V1
-  alias Kazan.Models.Apis.Extensions.V1beta1
-  alias Kazan.Models.Apis.Rbac
```

Comment:
As well as fixing the namespacing, I changed the aliases to not include the version as this makes the code clearer.  e.g. `%Extensions.V1beta1.Deployment` instead of  `%V1beta1.Deployment{}`